### ISSUE TITLE
Reverse Proxy setup

### ISSUE THREAD

**Issue by neuman1812** [morpheus65535/bazarr#184]
According to the Wiki for reverse proxy, the nginx code would be 

```
location /bazarr {
        proxy_pass              http://127.0.0.1:6767/bazarr;
```
However I was unable to get this to work.  It returned a 404 error. 
I noticed in the setting for the base-url,  when adding   /bazarr   it defaults with the trailing /   

![bazarr](https://user-images.githubusercontent.com/9267886/46879975-76595080-ce15-11e8-8661-e45b1ae51df3.png)

So I modified my Nginx config  with the following code:

```
    location /bazarr/ {
        proxy_pass              http://127.0.0.1:6767;
        proxy_set_header        X-Real-IP               $remote_addr;
        proxy_set_header        Host                    $host;
        proxy_set_header        X-Forwarded-For         $proxy_add_x_forwarded_for;
        proxy_set_header        X-Forwarded-Proto       $scheme;
     }
```

Not sure if this is an issue specific to my setup or not.  Just thought I'd add it.

**Comment by morpheus65535** [comment]
You are right, url in browser need to include the trailing slash. That's not related to reverse proxy, you'll get the same result if you access it directly.

**Comment by pcdevil** [comment]
Hello,   
I am using a similar setup. I have a docker-compose.yml file with NGINX and Bazarr with shared network and mappings to different ports.

I setup the base url in Bazarr and configured my NGINX as the [Bazarr's wiki page](https://github.com/morpheus65535/bazarr/wiki/Reverse-proxy) says.

Unfortunately when I tried to open it (with port in the url) it kept redirecting me to the same address but without port specification. 

I did some digging and came upon this answer on Server Fault: https://serverfault.com/a/731985

So basically replaced the `$host` to `$http_host` and it works like magic!
```
    location /bazarr/ {
        proxy_pass              http://bazarr:6767/bazarr/;
        proxy_set_header        X-Real-IP               $remote_addr;
        proxy_set_header        Host                    $http_host;
        proxy_set_header        X-Forwarded-For         $proxy_add_x_forwarded_for;
        proxy_set_header        X-Forwarded-Proto       $scheme;
     }
```

I suggest to update the wiki guide with this information!

**Comment by pcdevil** [comment]
Sure thing!

Let's say NGINX port is exposed to `8080` in the Docker compose config and the host is `example.com`, so the address for Bazarr would be `http://example.com:8080/bazarr/`.
When I entered it on the browser it resolved the address but immediately redirected me to `http://example.com/bazarr/`. The 80 port was not even open on the server so it failed fulfill the request. I saw the redirect with cURL as well.

In the discussion on Server Fault they mention the reason of the error is the application is not using the redirect properly. I am not sure it was Bazarr's fault, maybe it was NGINX but I used the same config but with `$host` for Sonarr and it worked fine.
